### PR TITLE
Fix warning when stais don't have a target

### PR DESCRIPTION
**Pull request by ghost**
This might happen even unintendedly during startup or shutdown.

@miv-ableton @toh-ableton 

Thanks @fnx-ableton for reporting this!

### CODE REVIEW

File: qml/Aqt/Cassowary/Stay.qml [hunk 1]
```diff
@@ -23,6 +23,6 @@ import Aqt.Cassowary 1.0
 
 Edit {
     id: edit
-    suggested: actualTarget.value
+    suggested: actualTarget ? actualTarget.value : NaN
```
Comment (fnx-ableton) [inline qml/Aqt/Cassowary/Stay.qml:26]: Oooh… `NaN` is a valid `double`? Good to know.

Comment (ghost) [inline qml/Aqt/Cassowary/Stay.qml:26]: Yes, it is here. On the other hand, you should not use NaN in `expr:`... It is easy to fix though now that I think about it, so I'm gonna fix that later too...

Comment (fnx-ableton) [inline qml/Aqt/Cassowary/Stay.qml:26]: Why shouldn't you…?

Comment (ghost) [inline qml/Aqt/Cassowary/Stay.qml:26]: It gets into the solver and then it starts doing weird things. Same happens with `Infinity` (we found out through experience to our disgrace...)

Comment (fnx-ableton) [inline qml/Aqt/Cassowary/Stay.qml:26]: :sob:

**Comment by toh-ableton** [comment]
lgtm

**Pull request merged**